CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Changes since v2.12
 - Long attachment filenames are now truncated in the UI (#2118)
 - `/api/applications/export` now doesn't blow up when an application has multiple forms. Instead only answers for the requested form are returned. (#2153)
 - Sort applications based on the application external id by sequence (#2183)
+- Re-enabling a catalogue item after its form was changed now works. (#2283)
```

Comment:
I think this is difficult to read :smile:

Reply:
woops, I've also added it to the wrong release!